gradle-sls-packaging/src/main/resources/init.sh

```diff
@@ -42,15 +42,22 @@ STATIC_LAUNCHER_CHECK_CONFIG="service/bin/launcher-check.yml"
 
 DEPRECATION_MESSAGE="Command is deprecated: the next major release of sls-packaging will only support start/status/stop"
 
+function print_help() {
```

Comment:
I don't think the name is correct here. This simply proxies args to `GO_INIT_CMD`

Reply:
It now just prints the help header.